Source/Extensions/UITextFieldExtension.swift

```diff
@@ -1,16 +1,16 @@
 import UIKit
 
 extension UITextField {
-
+    // Disable hardware keyboards.
     static func disableHardwareLayout() {
 #if targetEnvironment(simulator)
-        // Disable hardware keyboards.
+// swiftlint:disable indentation_width
         let setHardwareLayout = NSSelectorFromString("setHardwareLayout:")
 
         UITextInputMode.activeInputModes
-        // Filter `UIKeyboardInputMode`s.
-            .filter({ $0.responds(to: setHardwareLayout) })
+            .filter { $0.responds(to: setHardwareLayout) }
             .forEach { $0.perform(setHardwareLayout, with: nil) }
+// swiftlint:enable indentation_width
```

Comment:
I'm confused why we need to disable that rule? I can't see anything that would justify an exception?

Also, you can use 
```swift
// swiftlint:disable:next
```
to disable only next occurrence (saving you from having to close the disable start.